mungegithub/mungers/sig-mention-handler.go

```diff
@@ -18,22 +18,31 @@ package mungers
 
 import (
 	"fmt"
+	"regexp"
 	"strings"
 
-	"github.com/golang/glog"
 	"k8s.io/kubernetes/pkg/util/sets"
 	"k8s.io/test-infra/mungegithub/features"
 	"k8s.io/test-infra/mungegithub/github"
 	"k8s.io/test-infra/mungegithub/options"
+
+	"github.com/golang/glog"
+	githubapi "github.com/google/go-github/github"
 )
 
-var labelPrefixes = []string{"sig/", "committee/", "wg/"}
+var (
+	labelPrefixes = []string{"sig/", "committee/", "wg/"}
+	sigMentionRE  = regexp.MustCompile(`@\S+\nThere are no sig labels on this issue.`)
```

Comment:
I think you need a space between the `\n` and `There`.

Reply:
Do I? I don't see a space in the code: https://github.com/kubernetes/test-infra/blob/b37e88ba2570935fcdd0195c652a77bf42dd94df/mungegithub/mungers/sig-mention-handler.go#L114-L115

Reply:
I am the author of the handler. I confirmed there is no space.

Reply:
You're right. There is no space, but when I copy the text from github I get one anyways :stuck_out_tongue_closed_eyes: 
/approve

Reply:
i see! haha :)